left_wheel_names  : ['wheel_left_joint']
right_wheel_names : ['wheel_right_joint']

# In ROS1 these physical properties were parsed from the URDF
wheel_radius: 0.0985
wheel_separation: 0.4044

publish_rate: 50.0
pose_covariance_diagonal : [0.001, 0.001, 0.001, 0.001, 0.001, 0.01]
twist_covariance_diagonal: [0.001, 0.001, 0.001, 0.001, 0.001, 0.01]

# Base frame_id
base_frame_id: base_footprint
velocity_rolling_window_size: 10
